### PR TITLE
Webhook cleanup

### PR DESCRIPTION
**Pull request by abays**
Cleans up the webhook mechanism based on the latest improvements to the pattern used across our operators.

### CODE REVIEW

File: api/v1beta1/ovndbcluster_types.go [hunk 1]
```diff
@@ -28,6 +28,13 @@ const (
 	NBDBType = "NB"
 	// SBDBType - Southbound database type
 	SBDBType = "SB"
+
+	// Container image fall-back defaults
+
+	// OvnNbContainerImage is the fall-back container image for OVNDBCluster NB
+	OvnNbContainerImage = "quay.io/podified-antelope-centos9/openstack-ovn-nb-db-server:current-podified"
```
Comment (karelyatin) [inline api/v1beta1/ovndbcluster_types.go:35]: would be good to name these as OvnNBContainerImage for consistency

File: api/v1beta1/ovndbcluster_types.go [hunk 2]
```diff
@@ -28,6 +28,13 @@ const (
...
+	// OvnSbContainerImage is the fall-back container image for OVNDBCluster SB
+	OvnSbContainerImage = "quay.io/podified-antelope-centos9/openstack-ovn-sb-db-server:current-podified"
```
Comment (karelyatin) [inline api/v1beta1/ovndbcluster_types.go:37]: OvnSBContainerImage

Comment (abays) [reply api/v1beta1/ovndbcluster_types.go]: Done

File: api/v1beta1/ovndbcluster_types.go (hunk 1 above)
Comment (abays) [reply api/v1beta1/ovndbcluster_types.go]: Done

**Review by karelyatin — APPROVED** [review]
/lgtm

**Pull request merged**